src/pages/roadmap/Section.tsx

```diff
@@ -0,0 +1,56 @@
+import clsx from 'clsx'
+import { StyledLink } from 'src/components/StyledLink'
+import styles from './Section.module.scss'
+
+type SectionProps = {
+  title: React.ReactElement
+  icon: string
+  iconSize: {
+    with: string
+    height: string
+  }
+  iconPosition: 'left' | 'right'
+  subtitle: string
+  descriptionText: string
+  descriptionLink: string
+  projects: string[]
+  className?: string
+}
+
+export const Section: React.FC<SectionProps> = ({
+  title,
+  icon,
+  iconSize,
+  iconPosition,
+  subtitle,
+  descriptionText,
+  descriptionLink,
+  projects,
+  className,
+}) => {
+  return (
+    <div className={clsx(styles.container, iconPosition === 'right' && styles.reverse, className)}>
+      <div className={styles.iconWrapper}>
+        <div className={styles.icon}>
+          <img src={icon} alt="" style={{ width: iconSize.with, height: iconSize.height }} />
```

Comment:
Use this solution can avoid having to use `!important`.

```
const isMobile = useIsMobile()
<... style={isMobile ? { width: 184, height: 184 } : { width: iconSize.with, height: iconSize.height }} />
```

Reply:
updated